docs/docs/quickstart.md

```diff
@@ -1,5 +1,5 @@
----
-id: quickstart
----
+______________________________________________________________________
+
+## id: quickstart
```

Comment:
What caused this? its not correct

Reply:
Oh boy it's the https://mdformat.readthedocs.io/en/stable/users/style.html one - it only uses ATX headings and do not recognise frontmatter - also this one overwrites the 180 lines

Can we remove this pre-commit? it is going to cause a whole lot of issues.

Reply:
Sure. I will do it.